gym/lib/gym/runner.rb

```diff
@@ -155,14 +167,33 @@ def move_ipa
       ipa_path
     end
 
+    # copys framework from temp folder:
+
+    def copy_files_from_path(path)
+      UI.success "Exporting Files:"
+      framework_path = Dir[path]
+      framework_path.each do |f|
+        existing_file = File.join(File.expand_path(Gym.config[:output_directory]), File.basename(f))
+        # If the target file already exists in output directory
+        # we have to remove it first, otherwise cp_r fails even with remove_destination
+        # e.g.: there are symlinks in the .framework
+        if File.exist?(existing_file)
+          UI.important "Removing #{File.basename(f)} from output directory" if $verbose
+          FileUtils.rm_rf(existing_file)
```

Comment:
tbh. i was suprised by the bad beahviour on `cp_r` when there are symlinks in and subfolder. if the subfolder already exists they where not overwriteable.  i even tried to shell out, but osx itself has problems with recursive copy and symlink dereferencing. i found a few ruby hacks, that use `find` - but i think thats way too ugly.

the current solution uses:
`existing_file = File.join(File.expand_path(Gym.config[:output_directory]), File.basename(f))`

whereas there is a small chance for an edge-case where the user sets output_directory to `/` (not sure if this even works on osx - without some kernel params) - BUT `f` should never be empty, as `f`  is a result of the each loop. i also think - that of other FS restricitions there should be no chance that `f`  could be `*,~` or some other system relevant value.

let me know if you have another opinion - to further improve it!